mkdocs.yml

```diff
@@ -38,4 +38,8 @@ nav:
         - MEG systems: 99-appendices/07-meg-systems.md
         - Coordinate-systems: 99-appendices/08-coordinate-systems.md
       - Changelog: CHANGES.md
-    - The BIDS Starter Kit: https://github.com/bids-standard/bids-starter-kit
+    - The BIDS Starter Kit:
+      - GitHub repository: https://github.com/bids-standard/bids-starter-kit
+      - Tutorials: https://github.com/bids-standard/bids-starter-kit/wiki/Tutorials
+      - Forum: https://neurostars.org/tags/bids
```

Comment:
```suggestion
```

I think the neurostars forum is now sufficiently advertise (and framed in a better way) in #279

Reply:
I think that this is still fine to publicize here because it is part of the starter kit and probably better to keep them together as a group

Reply:
In how far is Neurostars a part of the BIDS-starter-kit? I see them as separate entities and think that combining them here will lead to confusion between the two.

Reply:
Regarding part of the starter kit - I think it is a good feature to publicize that we are using this. I agree that they are separate entities, but I think helpful to bring attention to this great resource. Several of us are active on NeuroStars and good to further promote as another resource for those to learn and pose questions. I think from the perspective of learning and starting out, putting the NeuroStars forum is useful to be there. Also depending on the conclusion of https://github.com/bids-standard/bids-specification/issues/189 may make it more of a need to add it in. 

To clarify the concern - the confusion can be that NeuroStars is for all neuroimaging questions and not just BIDS related questions?

Reply:
I think the concern is simply that the BIDS Starter Kit is a related set of documents, and NeuroStars is a useful resource, and the way this is currently written makes it appear that NeuroStars is a sub-structure of the starter kit.

I am +1 for removal, myself.

Reply:
I agree, in the current setup that can be misunderstood as solely part of the BIDS. 

Perhaps moving it to be it's own button? To clarify - the starter kit would have the drop down menu then under this (at the same level as the starter kit) have the NeuroStars forum to directly click and direct to our BIDS tag on NeuroStars. I think drawing attention to this forum is helpful.

Reply:
I think it's okay to advertise the BIDS starter kit on the specification - and I saw it as a compromise between not putting too many links to other sites onto the specification and yet still doing a *bit* of advertising.

I agree that Neurostars is a great resource, perhaps we can advertise it on the starterkit Wiki page, which is a spot we are linking to.

Reply:
We can experiment with this setup (removing NeuroStars) and iterate from there. The starter kit does link out to NeuroStars. My concern is missing the opportunity to provide the most help and resources to get them into the environment but can experiment and learn